guides/doc-Content_Management_Guide/topics/Using_ISS.adoc

```diff
@@ -26,7 +26,39 @@ You cannot use ISS to synchronize content from {ProjectServer} to {SmartProxySer
 {SmartProxyServer} supports synchronization natively.
 For more information, see {PlanningDocURL}chap-Documentation-Architecture_Guide-Capsule_Server_Overview[{SmartProxyServer} Overview] in _Planning for {ProjectNameX}_.
 
-==== Air gapped
+=== Configuration Definitions
+There are different ways of using ISS to synchronize content between two {ProjectServer}s.
+
+.Connected
+In a connected scenario, two {ProjectServer}s are connected and are able to communicate with each other.
```

Comment:
The second 'connected' here is slightly redundant. How about `in a connected scenario two $servers are able to communicate with each other over a network.